sbom/mocks/scanner.go

```diff
@@ -0,0 +1,102 @@
+// Code generated by mockery v2.53.3. DO NOT EDIT.
```

Comment:
Where is this file coming from? Is this related to this pr?

Reply:
It is auto generated. The `go generate` command can be run to run code generators, which we use to run mockery. What it generates is dictated by comments in the code like [this one](https://github.com/paketo-buildpacks/libpak/pull/408/files#diff-c624765f4df697256be192f9789ec38a15b70661f77956b29d03c81856fa8414R91).

Reply:
This one surprised me. It looks like someone, probably me, forgot to run `go generate` in the past and we never generated the mock for this Interface.